particle-iot/device-os
# Fix msom usb + ble setup names

### Problem
Msom setup shows the wrong name: `Mseries`. It should be `M SoM`
Same for P2 BLE setup. 

### Solution
Fix the hardcoded USB + BLE setup strings

### Steps to Test
Build + flash branch. Setup msom on setup.particle.io, name should show as `M SoM`

### Example App
tinker

### References

Links to the Community, Docs, Other Issues, etc..

---

### Completeness
